source/device/image_client.py

```diff
@@ -80,7 +81,13 @@ def _deserialize_image_chunk(data):
 			results.setdefault(counter, []).append(header)
 			# append image
 			image_hex = data[header['HEADER_SIZE']:header['CHUNK_SIZE']]
-			image = mpimg.imread(io.BytesIO(image_hex), format='jpg')
+			if header['CHUNK_TYPE'] == 260:
+				image = mpimg.imread(io.BytesIO(image_hex), format='jpg')
+			elif header['CHUNK_TYPE'] == 251:
```

Comment:
Do we want public to know what 260 and 251 is? If yes I would add some statics.

Reply:
Good idea, maybe I'll do that already, who knows which types will get supported in the future.